src/api/spec/support/files/service

```diff
@@ -0,0 +1,14 @@
+<services>
```

Comment:
Right! I will rename the file. And yes it's not necessary to have a full file, in fact, we don't need the file, so I will remove it and add a little content.